releases/release-1.21/release-notes-draft.md

```diff
@@ -49,47 +58,65 @@
 
 ### Documentation
 
-- Official support to build kubernetes with docker-machine / remote docker is removed. This change does not affect building kubernetes with docker locally. ([#97935](https://github.com/kubernetes/kubernetes/pull/97935), [@adeniyistephen](https://github.com/adeniyistephen)) [SIG Release and Testing]
```

Comment:
where did this go?

Reply:
I think it was de-deduplicated because https://github.com/kubernetes/kubernetes/pull/97618 has the same release-note

Reply:
interesting, but I can't find 97618 in this doc either?

Reply:
oh nevermind, found it -- was searching on master instead of the PR